test/test.js

```diff
@@ -185,3 +185,74 @@ describe('Setup custom settings', function() {
 		).equal('ns-block~~element ns-block~~element--mod-value');
 	});
 });
+
+describe('From class map', function() {
```

Comment:
`'From class map'` → `'Block with class mapping'`?
Because of `Block with class mapping should return given block name`...